nbs/cli.ipynb

```diff
@@ -60,6 +60,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    #     nbdev_new.__wrapped__(repo='my-project', user='fastai', author='fastai', author_email='info@fast.ai', description='A test project')
</pre><p>Do you think this cell can be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1009/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I added it as a convenience while dev’ing. It might eventually become a test